source/includes/openshift/deploy-minio-on-openshift.rst

```diff
@@ -136,19 +136,165 @@ The web console displays a widget for tracking the installation progress.
 
 Once installation completes, click :guilabel:`View Operator` to view the MinIO Operator page. 
 
-3) Open the MinIO Operator Interface
+3) Configure TLS Certificates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have installed the MinIO Operator from Red Hat OperatorHub, the installation process also configures the :openshift-docs:`OpenShift Service CA Operator <security/certificate_types_descriptions/service-ca-certificates.html>`.
+This Operator manages the the TLS certificates required to access the MinIO Operator Console and Tenants.
+It automatically renews and rotates the certificates 13 months before expiration.
+No additional action is required.
+
+For Operator installations deployed by other methods, configure the :openshift-docs:`Service CA certificates <security/certificate_types_descriptions/service-ca-certificates.html>` manually.
+See the dropdowns below for details.
+
+.. dropdown:: OpenShift Service CA Certificate configuration
+
+   To manually enable the ``service-ca`` Operator to manage TLS certificates:
+
+   #. Use the following command to edit the deployment:
+
+      .. code-block:: shell
+         :class: copyable
+
+         oc edit deployment minio-operator  -n minio-operator
```

Comment:
`oc edit` opens the correct file without having to hunt around for it. Very handy.

Reply:
Oh should add mention of what `oc` is